frontend/public/locales/en/common.json

```
@@ -0,0 +1,219 @@
+  "home_spotlight_description": "Get to home page",
+  "clubs_spotlight_description": "View, add or delete clubs",
+  "user_settings_title": "User Settings",
+  "user_settings_spotlight_description": "Change name, email, password etc.",
+  "planning_title": "planning",
+  "planning_spotlight_description": "Change planning of matches",
+  "teams_title": "Teams",
+  "teams_spotlight_description": "View, add or delete teams",
+  "players_title": "Players",
+  "players_spotlight_description": "View, add or delete players",
+  "stage_title": "Stage",
+  "stage_spotlight_description": "Change the layout of the tournament",
+  "court_title": "Courts",
+  "court_spotlight_description": "View, add or delete courts",
+  "tournament_setting_title": "Tournament Settings",
+  "tournament_setting_spotlight_description": "Change the settings of the tournament",
+  "nothing_found_placeholder": "Nothing found ...",
+  "search_placeholder": "Search ...",
+  "at_least_one_team_validation": "Enter at least one team",
+  "active_teams_checkbox_label": "These teams are active",
+  "team_name_input_placeholder": "Best Team Ever",
+  "active_team_checkbox_label": "This team is active",
+  "team_member_select_placeholder": "Pick all that you like",
+  "team_member_select_title": "Team members",
+  "single_team": "Single Team",
+  "multiple_teams": "Multiple Teams",
+  "add_team_button": "Add Team",
+  "edit_team_title": "Edit Team",
+  "club_select_label": "Club",
+  "now_button": "NOW",
+  "logout_title": "Logout",
+  "user_title": "User",
+  "more_title": "More",
+  "website_title": "Website",
+  "github_title": "Github",
+  "api_docs_title": "API docs",
+  "elo_input_label": "Max ELO difference",
+  "only_recommended_input_group_label": "Only show teams that played less matches",
+  "only_recommended_radio_label": "Only recommended",
+  "all_matches_radio_label": "All matches",
+  "max_results_input_label": "Max results",
+  "iterations_input_label": "Iterations",
+  "delete_club_button": "Delete Club",
+  "title": "Title",
+  "name_field_text": "name",
+  "delete_court_button": "Delete Court",
+  "win_distribution_text_win": "wins",
+  "win_distribution_text_draws": "draws",
+  "win_distribution_text_losses": "losses",
+  "delete_player_button": "Delete Player",
+  "status": "Status",
+  "created": "Created",
+  "elo_score": "ELO score",
+  "swiss_score": "Swiss score",
+  "name_table_header": "Name",
+  "member_table_header": "Members",
+  "active": "Active",
+  "inactive": "Inactive",
+  "delete_team_button": "Delete Team",
+  "delete_tournament_button": "Delete Tournament",
+  "edit_tournament_button": "Edit Tournament",
+  "recommended_badge_title": "Recommended",
+  "upcoming_matches_empty_table_info": "upcoming matches",
+  "elo_difference": "ELO Difference",
+  "swiss_difference": "Swiss Difference",
+  "could_not_find_any_alert": "Could not find any",
+  "dropzone_accept_text": "Drop files here",
+  "dropzone_reject_text": "PDF file must be less than 10MB",
```

```suggestion
  "dropzone_reject_text": "Image must be less than 10MB",
```
Oh this was already written incorrectly by me before. It's not about PDFs, but images.
could you fix the Chinese translation as well?